cmd/quadlet/main.go

```diff
@@ -13,6 +14,7 @@ import (
 	"github.com/containers/podman/v4/pkg/systemd/parser"
 	"github.com/containers/podman/v4/pkg/systemd/quadlet"
 	"github.com/containers/podman/v4/version/rawversion"
+	"github.com/sirupsen/logrus"
```

Comment:
This will increase the binary size of Quadlet.  Can we avoid logrus and rather print to STDERR?